pkg/arvo/app/contact-view.hoon

```diff
@@ -40,9 +46,7 @@
     :_  this
     :~  [%pass /updates %agent [our.bowl %contact-store] %watch /updates]
         (contact-poke:cc [%create /~/default])
-        (group-poke:cc [%bundle /~/default])
         (contact-poke:cc [%add /~/default our.bowl *contact])
-        (group-poke:cc [%add [our.bowl ~ ~] /~/default])
```

Comment:
Why are you removing the default group?

Reply:
What purpose does it serve now?

Reply:
It’s a placeholder for ad-hoc contact sharing. Perhaps better served by a TODO